state/azure/cosmosdb/cosmosdb.go

```diff
@@ -481,7 +480,7 @@ func createUpsertItem(contentType string, req state.SetRequest, partitionKey str
 		isBinary = false
 	}
 
-	ttl, err := parseTTL(req.Metadata)
+	ttl, err := stateutils.ParseTTL(req.Metadata)
```

Comment:
Note that this method ("parseTTL") was common among many components, so I've moved it to a shared package

Reply:
Prefer using https://github.com/dapr/components-contrib/blob/master/metadata/utils.go#L58 `TryGetTTL`

Reply:
Responded below. Don't think that's possible